### PR TITLE
2 - Centrailized, reorganized CMake configuration and resources

### PR DESCRIPTION
**Pull request by redtide**
- Put all configurations in a single place for CMake
- Restructured resource files and folders
- Ordered view in QtCreator project tree

see also #6

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,16 +1,122 @@
...
-
-find_package(Qt5 CONFIG REQUIRED Core Widgets)
-find_package(KF5GuiAddons CONFIG REQUIRED)
-find_package(KF5WidgetsAddons CONFIG REQUIRED)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Woverloaded-virtual")
+set(CMAKE_CXX_STANDARD 17) #Qt6 needs C++17
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+#===============================================================================
+# Qt
+#===============================================================================
```
Comment (agateau) [inline CMakeLists.txt:13]: nit: I prefer not to have banner-style comments as I find them tedious to maintain. I prefer something simpler, like this (note the starting empty line):
```suggestion

#
# Qt
#
```

**Review by agateau — CHANGES_REQUESTED** [review]
Minor remarks, but this looks good!

Can you rebase it to remove the unrelated "HiDPI support" commit?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Colorpick: Color picker and contrast checker
 
-![Contrast check](screenshots/contrast.png)
+![Contrast check](resources/screenshots/contrast.png)
```
[the remark this answers is not in the document]
Comment (redtide) [reply README.md:3]: It's not shipped but can be used as "external" resource: it is referenced by a future "/usr/share/metainfo/colorpick.appdata.xml", at least I consider also external resources, not just "files built in the application".
Just saying, I'll leave as is.

**Comment by redtide** [comment]
There is some mess in my current git history, I'm not sure how to rebase with the unmerged hidpi commit, so I'll wait you to merge it and the ignore files PR and continue from there.

**Comment by redtide** [comment]
Should be OK now.

**Review by agateau — APPROVED** [review]
Looks good now!

**Pull request merged**